package.json

```diff
@@ -74,6 +75,7 @@
     "rollup-plugin-filesize": "6.1.0",
     "rollup-plugin-node-resolve": "5.0.1",
     "rollup-plugin-resolve": "0.0.1-predev.1",
+    "rollup-plugin-flow": "^1.1.1",
```

Comment:
Great PR, but could we remove this flow plugin and then changes in the rollup file?

What are they doing, and if they are needed, could we have them in a new PR?

Reply:
This is because the flow type is described in `@react-native-community/async-storage` without an explanation of`@flow`.
Without this plug-in, rollup can not be performed and can not be built, so it is included in this PR.

It is good to cut out and create another PR, what does that mean?

Reply:
Ah ok that makes sense. I didn't know about that, but I never had a problem with that with typescript. If it's needed for this repo, sure! 👍

Reply:
Like this file
https://github.com/react-native-community/async-storage/blob/104bfef9fb/lib/hooks.js

Reply:
@pvinis

Reply:
thanks!

Reply:
@nitaking I would be more inclined to see if we could get this fix upstream instead of us having to solve it in this lib. What would the fix actually be? Just adding `@flow` to the linked file in the async-storage lib?

Reply:
👍